File: CS50/pset6/mario/mario.py
from cs50 import get_int

while True:
    height = get_int("Height : ")
    if height < 24 and height >= 0:
        for i in range(height):
            print((height - (i+1)) * " ", end="")
            print("#" * (i + 2), end = "")
            print()
        break

File: CS50/pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height = 0;
    //ask for the height of the piramid to the user
    do
    {
        printf("Height : ");
        height = get_int();
    } while (height < 0 || height > 23);
    // draw the piramid
   for(int i = 1; i < height + 1; i++)
   {
        for (int k=0;k<height-i;k++)
        {
            printf(" ");
        }
        for (int j=0;j<1+i;j++)
       {
           printf("#");
       }
       printf("\n");
   }
}